vulnerable_people_form/templates/_spl_match_email_template.md

```diff
@@ -1,18 +1,20 @@
 Dear {{first_name}} {{last_name}},
 
-Your registration number is {{reference_number}}.
+Your registration number is {{submission_id}}.
```

Comment:
has this changed ?

Reply:
Copy-paste error. It's changed in the backend only.